docs/README.md

```diff
@@ -110,30 +112,11 @@ The app launches a map view containing the pre-configured web map.
 |-----|-----|
 | ![Main Map View](/docs/images/anatomy-map-view-uwp.png) | ![Main Map View](/docs/images/anatomy-map-view.png) |
 
-The navigation bar's title reflects the name of the web map and the navigation bar button items are as follows:
-
-| Icon | Description |
-| ---- | ----------- |
-| ![Bookmarks icon](/src/DataCollection.UWP/Assets/Calcite/bookmark-32.png) | Bookmarks button reveals list of bookmarks in the map |
-| ![TOC icon](/src/DataCollection.UWP/Assets/Calcite/layers-32.png) | TOC button shows a list of layers, which can be toggled on and off |
-| ![Legend icon](/src/DataCollection.UWP/Assets/Calcite/legend-32.png) | Legend button shows the layer symbols for the map |
-| ![Hamburger Drawer View](/src/DataCollection.UWP/Assets/Calcite/ellipsis-32.png) | Ellipsis button to reveal or hide the app context drawer view. |
-| ![Zoom To Location](/src/DataCollection.UWP/Assets/Calcite/gps-on-32.png) | Zoom to user's location. |
-| ![Add Feature](/src/DataCollection.UWP/Assets/Calcite/plus-circle-32.png) | Add a new spatial feature to map. |
-
-### Manage the app's context
```

Comment:
Looks like deletion of this heading threw off the TOC structure, but I'm happy to regenerate it since I already have the TOC generator extension installed